unix_integration/common/src/unix_config.rs

```diff
@@ -1,13 +1,15 @@
+use std::env;
```

Comment:
This module needs documentation so it's understandable without picking through it all to work out what things are used for. tThere's a kanidmconfig, unixdconfig and configint, and even knowing the history well, I'm starting to draw diagrams just to work out how it all comes together.

Reply:
Also every config file field needs to have a `///` documentation so it's clear what everything does, and it'd be good if they were pub so it's easier to link to in the docs for reference, rather than digging into source code.